product-scenarios/scenarios-common/pom.xml

```diff
@@ -94,5 +94,9 @@
             <groupId>io.swagger</groupId>
             <artifactId>swagger-models</artifactId>
         </dependency>
+        <dependency>
```

Comment:
why is this dependency needed?

Reply:
This dependency is used in Webapp deployment